.NET/Microsoft.Recognizer.Text.TimeZone/English/TimeZoneParser.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Text.RegularExpressions;
+using Microsoft.Recognizers.Definitions.English;
+
+namespace Microsoft.Recognizers.Text.TimeZone.English
+{
+    class TimeZoneParser : IParser
+    {
+        public TimeZoneParser()
+        {
+
+        }
+        public int getMinutes(string matched)
+        {
+            int bs = 1;
+            if (matched.StartsWith("+")
```

Comment:
Nitpick: don't break small expressions, and if you do, please keep the operator at the end of the line (i.e. ||).